libs/libc/string/lib_strncasecmp.c

```diff
@@ -50,4 +50,9 @@ int strncasecmp(const char *cs, const char *ct, size_t nb)
 
   return result;
 }
+
+int strncasecmp_l(const char *cs, const char *ct, size_t nb, locale_t l)
+{
+  return strncasecmp(cs, ct, l);
```

Comment:
```suggestion
  return strncasecmp(cs, ct, nb);
```
Wrong parameter passed. CI build is failing due to this error.

Reply:
Done.